src/main/webapp/app/exercises/programming/manage/programming-exercise.component.html

```diff
@@ -180,32 +173,30 @@ <h4 class="mb-1 me-auto">
                                     <span class="d-md-inline" jhiTranslate="artemisApp.exercise.teams">Teams</span>
                                 </a>
                             </div>
-                            <div class="btn-group-vertical me-1 mb-1">
-                                <a
-                                    *ngIf="programmingExercise.isAtLeastTutor"
-                                    [routerLink]="['/course-management', courseId, 'programming-exercises', programmingExercise.id, 'participations']"
-                                    class="btn btn-primary btn-sm me-1 mb-1"
-                                    style="display: flex; justify-content: center; align-items: center"
-                                >
-                                    <div>
-                                        <fa-icon [icon]="faListAlt"></fa-icon>
-                                        <span class="d-md-inline" jhiTranslate="artemisApp.exercise.participations">Participations</span>
-                                    </div>
-                                </a>
+                            <div class="btn-group-vertical me-1">
                                 <a
                                     [jhiFeatureToggleLink]="FeatureToggle.ProgrammingExercises"
                                     *ngIf="programmingExercise.isAtLeastEditor"
                                     [routerLink]="['/course-management', courseId, 'programming-exercises', programmingExercise.id, 'grading', 'test-cases']"
-                                    class="btn btn-info btn-sm me-1 mb-1"
+                                    class="btn btn-warning btn-sm me-1 mb-1"
                                     style="display: flex; justify-content: center; align-items: center"
                                 >
                                     <div>
-                                        <fa-icon [icon]="faFileSignature"></fa-icon>
+                                        <fa-icon [icon]="faUserCheck"></fa-icon>
                                         <span class="d-md-inline" jhiTranslate="artemisApp.programmingExercise.configureGrading.shortTitle">Grading</span>
                                     </div>
                                 </a>
+                                <a
+                                    *ngIf="programmingExercise.isAtLeastEditor"
+                                    [routerLink]="['/course-management', programmingExercise.course!.id, 'programming-exercises', programmingExercise.id, 'exercise-hints']"
+                                    class="btn btn-warning btn-sm me-1 mb-1"
+                                    style="margin-bottom: 10px"
```

Comment:
According to my devtools, this style is never actually applied (at least not in the normal exercise lists). Probably it can just be removed?